client/layout/masterbar/style.scss

```diff
@@ -38,7 +43,7 @@ $autobar-height: 20px;
 	display: flex;
 		align-items: center;
 	position: relative;
-	color: $white;
+	@include theme( color, --masterbar-color );
```

Comment:
Why do we need an include? Can't we just use var assignment? We should strive to have _one and only one way to define colors_ unless there are clear technical needs (in which case we need to assess if it's really beneficial).

```
color: var( --masterbar-color  );
```